pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -72,6 +72,7 @@ let
         knm            = versionAtLeast ccVersion "8.0";
         # AMD
         znver1         = versionAtLeast ccVersion "6.0";
+        znver2         = versionAtLeast ccVersion "9.0";
```

Comment:
I am not sure when `znver2` support has been added no `gcc` and `clang`, so these version numbers are not final yet